install/calibre-server-install.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024
+# Author: thisisjeron
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y imagemagick xvfb libxcomposite1
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Calibre (latest)"
+# If your container runs as root, you generally do not need to prefix with `sudo`.
+# The official Calibre instructions:
+wget -nv -O- https://download.calibre-ebook.com/linux-installer.sh | sh /dev/stdin
```

Comment:
Before Downloading, can you get the Version number somehow? Just so you can check during the update that it is needed. And is this operation truly silent?